R/bundle.R

```diff
@@ -164,7 +164,12 @@ bundleApp <- function(appName, appDir, appFiles, appPrimaryDoc, assetTypeName,
                       contentCategory) {
 
   # infer the mode of the application from its layout
-  appMode <- inferAppMode(appDir, appPrimaryDoc, appFiles)
+  # unless we're an API, in which case, we're API mode.
+  if (contentCategory == "api") {
```

Comment:
I think you're right, I think I'll take your suggestion below instead though, so this check is getting removed.